app/assets/stylesheets/apply.scss.erb

```diff
@@ -284,3 +284,19 @@
 .student_applications-new, {
   textarea { height: em(135px); }
 }
+
+.applications-closed {
+  color: black;
+  background: #f5f5f5;
+  height: 100px;
+  width: 100%;
+  font-size: 90%;
+  padding-left: 10px;
+  padding-right: 10px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #E6E6E6;
```

Comment:
Maybe we can move these hex codes into `_variables.scss` and just reference them here?